### PR TITLE
New Pull RQ

### PR DESCRIPTION
**Pull request by julerobb1**

### CODE REVIEW

File: TroopAnnouncements.html [hunk 1]
```diff
@@ -65,15 +65,15 @@ <h2 class="w3-center">HTML slides</h2>
...
         
-          <img class="mySlides" src="https://media.discordapp.net/attachments/897388108591542304/1044398441981292634/b530938503bf4825a0a92b2f4be08296.png?width=393&height=1000" style="width:900%">
+          <img class="mySlides" src="https://imgur.com/qfDyUr0" style="width:900%">
         
           <div class="mySlides w3-container w3-xlarge w3-white w3-card-4">
-            <p><span class="w3-tag w3-red">New!</span>
-            <p>6 Crystal Glasses</p>
+            <p><span class="w3-tag w3-red">Merrit Bade University</span>
+            <p>We have been invited by  </p>
             <p>Only $99 !!!</p>
```
Comment (Byrdman32) [inline TroopAnnouncements.html]: > `<p>Only $99 !!!</p>`

This is not the right price for MBU.

File: TroopAnnouncements.html [hunk 2]
```diff
@@ -65,15 +65,15 @@ <h2 class="w3-center">HTML slides</h2>
...
             <img class="myslides" src="https://media.discordapp.net/attachments/897388108591542304/1044397019013332992/SPOILER_7a4062d7f4b1410ebb66a8414d1590ee.png?width=1617&height=1002">
```
Comment (Byrdman32) [inline TroopAnnouncements.html]: > ``

I don't think this is the right image to use here.

File: TroopAnnouncements.html [hunk 3]
```diff
@@ -65,15 +65,15 @@ <h2 class="w3-center">HTML slides</h2>
...
           </div>
         
-          <img class="mySlides" src="https://media.discordapp.net/attachments/897388108591542304/1044398441981292634/b530938503bf4825a0a92b2f4be08296.png?width=393&height=1000" style="width:900%">
+          <img class="mySlides" src="https://imgur.com/qfDyUr0" style="width:900%">
         
           <div class="mySlides w3-container w3-xlarge w3-white w3-card-4">
-            <p><span class="w3-tag w3-red">New!</span>
-            <p>6 Crystal Glasses</p>
+            <p><span class="w3-tag w3-red">Merrit Bade University</span>
+            <p>We have been invited by  </p>
```
Comment (Byrdman32) [inline TroopAnnouncements.html]: > `<p>We have been invited by </p>`

Incomplete sentence.

File: index.html [hunk 4]
```diff
@@ -187,42 +187,64 @@
...
+          <p class="author">-Main Exploration Bot v.1.17-</p>
+        </div>
+
+        <div class="mySlides">
+          <h3><b><center> If you would like volunteer to aid with cleanup please visit  </center></b><br><center>http://littlerock.gov/volunteer</center></h3>
+          <p class="author">-Main Exploration Bot v.1.17-</p>
+        </div>
+     
+      <div class="mySlides"> 
+        <h3><b><center></center></b><br><center>If you would like to donate to help with recovery efforts, text "GIVE" (without quotes) to 501 - 378 - 1111</center></h3>
```
Comment (Byrdman32) [inline index.html]: > If you would like to donate to help with recovery efforts, text "GIVE" (without quotes) to 501 - 378 - 1111

Where did you find this phone number? I can't find anywhere online that says this.

File: index.html [hunk 5]
```diff
@@ -187,42 +187,64 @@
...
+        <p class="author">-Main Exploration Bot v.1.17-</p>
+      </div>-->      
+
+      <div class="mySlides">
```
Comment (Byrdman32) [inline index.html:205]:
```<div class="mySlides">
        <h3><b><center> Troop outing on Jan 13 2023!</center></b><br><center>RSVP Via Scoutbook, See link for details https://imgur.com/qfDyUr0 </center></h3>
        <p class="author">-Main Exploration Bot v.1.17-</p>
      </div>
      <div class="mySlides">
        <h3><b><center> Troop outing on Jan 13 2023!</center></b><br><center>RSVP Via Scoutbook, See link for details https://imgur.com/qfDyUr0</center></h3>
        <p class="author">-Main Exploration Bot v.1.17-</p>
      </div>
      <div class="mySlides">
        <h3><b><center> Troop outing on Jan 13 2023!</center></b><br><center>RSVP Via Scoutbook, See link for details https://imgur.com/qfDyUr0 </center></h3>
        <p class="author">-Main Exploration Bot v.1.17-</p>
      </div>
      <div class="mySlides">
        <h3><b><center> Troop outing on Jan 13 2023!</center></b><br><center>RSVP Via Scoutbook, See link for details https://imgur.com/qfDyUr0 </center></h3>
        <p class="author">-Main Exploration Bot v.1.17-</p>
      </div>```


Remove old events.

**Review by Byrdman32 — CHANGES_REQUESTED** [review]
Please look through the comments I left and fix them or give an understanding of why we want to make these changes before I will approve this request.

File: index.html (hunk 4 above)
Comment (julerobb1) [reply index.html]: That was a typo, it was supposed to be the text number for thv11 : 501 376 111

File: TroopAnnouncements.html (hunk 1 above)
Comment (julerobb1) [reply TroopAnnouncements.html]: Im not sure where the 99 dollars came from.

File: TroopAnnouncements.html (hunk 2 above)
Comment (julerobb1) [reply TroopAnnouncements.html]: I agree. This was back when I was testing the slide thing and I apparently forgot to remove it. You may go ahead and remove it..half of the code for the troop announcement page is demo code from w3 schools..

File: TroopAnnouncements.html (hunk 3 above)
Comment (julerobb1) [reply TroopAnnouncements.html]: This was supposed to say something about a tour of nuclear one but I can no longer find the email.

File: index.html (hunk 4 above)
Comment (Byrdman32) [reply index.html]: > That was a typo, it was supposed to be the text number for thv11 : 501 376 111

Do you mean (501) 376 1111?

File: index.html (hunk 5 above)
Comment (julerobb1) [reply index.html:205]: Agreed, should be commented out

File: index.html (hunk 4 above)
Comment (julerobb1) [reply index.html]: Correct

**Review by Byrdman32 — APPROVED** [review]

File: TroopAnnouncements.html (hunk 3 above)
Comment (Byrdman32) [reply TroopAnnouncements.html]: I have removed this block

File: TroopAnnouncements.html (hunk 2 above)
Comment (Byrdman32) [reply TroopAnnouncements.html]: Image removed

File: TroopAnnouncements.html (hunk 1 above)
Comment (Byrdman32) [reply TroopAnnouncements.html]: Price segment removed.

**Pull request merged**